docs/anchoring-assets/overview.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Overview
+sidebar_position: 10
+---
+
+import { CodeExample } from "@site/src/components/CodeExample";
+
+## What is an anchor?
+
+An anchor is a Stellar-specific term for the on and off-ramps that connect the Stellar network to traditional financial rails, such as financial institutions or fintech companies. Anchors accept deposits of fiat currencies (such as the US dollar, Argentine peso, or Nigerian naira) via existing rails (such as bank deposits or cash-in points), then sends the user the equivalent digital tokens representing those deposits on the Stellar network. Alternatively, anchors allow token holders to redeem their tokens for the real-world assets they represent.
```

Comment:
Nowadays most of our anchors facilitate USDC transactions in exchange for nonequivalent fiat currencies, for example purchasing 10 USDC for 50 BRL.

In retrospect, using the terms "deposit" & "withdraw" may have been a mistake since it makes sense when the Stellar asset and fiat currency are equivalent but sounds off when used in the context of nonequivalent assets. Saying you deposited USDC into your Stellar account with Brazilian Reals sounds weird.

This is obviously a bigger change than this PR, but I've been starting to use the terms "buy" and "sell" in place of "deposit" and "withdraw", for what its worth.